### PR TITLE
Kube metrics script

### PR DESCRIPTION
**Pull request by yyyogev**
# What changed

Made a script to deploy Metricbeat to Kubernetes to make things easier

@imnotashrimp please review the string in quickstart.sh

----

## Pages to review

- https://deploy-preview-417--logz-docs.netlify.com/shipping/metrics-sources/kubernetes.html

## Remaining work

- [x] Technical review
- [x] Copy Review

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

**Comment by yyyogev** [comment]
This PR can replace https://github.com/logzio/logz-docs/pull/406

File: shipping-config-samples/quickstart.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
+has_kubectl=$(kubectl 2>&1)
+if [[ $has_kubectl == *"command not found"* ]]; then
+  echo "kubectl not found! please install and configure kubectl first"
```
Comment (imnotashrimp) [inline shipping-config-samples/quickstart.sh]:
```suggestion
  echo "kubectl not found! Please install and configure kubectl first."
```

File: shipping-config-samples/quickstart.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  kubectl --namespace=kube-system apply -f kube-state-metrics/examples/standard
+  rm  -rf kube-state-metrics
+fi
+
+kube_stat_ns=$(kubectl get deployments --all-namespaces -o json | jq '.items[] | select(.metadata.name == "kube-state-metrics")' | jq -r '.metadata.namespace')
+kube_stat_port=$(kubectl get deployments --all-namespaces -o json | jq '.items[] | select(.metadata.name == "kube-state-metrics")' | jq '.spec.template.spec.containers[0].ports[] | select(.name == "http-metrics")' | jq '.containerPort')
+
+read -esp "Enter your Logz.io Metrics Shipping Token:🔒" metrics_token
+printf "\n"
+read -ep "Enter your Logz.io region [us]:" logzio_region
```
Comment (imnotashrimp) [inline shipping-config-samples/quickstart.sh]:
```suggestion
read -ep "Logz.io region [us]: " logzio_region
```

File: shipping-config-samples/quickstart.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+if [[ ! -z $logzio_region ]] && [[ $logzio_region != "us" ]]; then
+  logzio_region="-${logzio_region}"
+else
+  logzio_region=""
+fi
+listener_host="listener${logzio_region}.logz.io"
+
+read -ep "Enter Kubelet shipping protocol [http]:" shipping_protocol
```
Comment (imnotashrimp) [inline shipping-config-samples/quickstart.sh]:
```suggestion
read -ep "Kubelet shipping protocol [http]: " shipping_protocol
```

File: shipping-config-samples/quickstart.sh [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+kubectl --namespace=kube-system create secret generic logzio-metrics-secret \
+  --from-literal=logzio-metrics-shipping-token=$metrics_token \
+  --from-literal=logzio-metrics-listener-host=$listener_host
+
+cluster_name=$(kubectl config current-context)
+if [[ $cluster_name == *"cluster/"* ]]; then
+  cluster_name=${cluster_name#*"cluster/"}
+fi
+read -ep "Enter your cluster name [${cluster_name}]:" real_cluster_name
```
Comment (imnotashrimp) [inline shipping-config-samples/quickstart.sh]:
```suggestion
read -ep "Cluster name [${cluster_name}]: " real_cluster_name
```

**Pull request merged**